killer_subscriber.go

```diff
@@ -0,0 +1,36 @@
+package signal
+
+import (
+	"log"
+	"os"
+	"syscall"
+)
+
+var signalsKiller = []os.Signal{os.Kill, os.Interrupt, syscall.SIGHUP, syscall.SIGTERM, syscall.SIGQUIT, syscall.SIGSTOP}
+var killerSubscriber func()
+
+func KillerSubscriber() func() {
+	if killerSubscriber == nil {
+		stopping := false
+		killerSubscriber = Subscribe(func(signal os.Signal) {
+			if signal == os.Kill {
+				log.Println("killing application")
+				os.Exit(1)
```

Comment:
http://tldp.org/LDP/abs/html/exitcodes.html

As I understand, I think we should return a >255 exit code here ...